pages/instructorScrapPaper/instructorScrapPaper.js

```diff
@@ -0,0 +1,52 @@
+const ERR = require('async-stacktrace');
+const express = require('express');
+const router = express.Router();
+const { createBarcodedPdf } = require('../../lib/scrapPaperMaker');
+const error = require('../../prairielib/lib/error');
+
+const pageLimit = 1000;
+const charLimit = 45;
+
+router.get('/', (req, res) => {
```

Comment:
Nope. This was missed.

Reply:
I'm looping back to this to discuss after the rest of this.

Reply:
![image](https://user-images.githubusercontent.com/7872295/147273719-a360e865-3df7-4f44-b6ab-1d0d2b760eb3.png)

Is there any easy way we want to manage this authentication? There is no middleware that authenticates someone based on having ANY instructor credentials in a course.

Reply:
These are the options as they appear. 

![image](https://user-images.githubusercontent.com/7872295/147273865-e9d7295c-101d-4412-a674-b64803d1a605.png)

I will look for an easy fix for the meantime, but this has to be thought out.

Reply:
Implemented in: 29a017708818fd1e90c5e6d5632880622609db8d

Reply:
Thanks! cc @jonatanschroeder @mwest1066 do you think `has_course_permission_edit` is the right thing to gate on? I know this doesn't yet handle the case of a CBTF proctor not affiliated with any one course trying to print scrap paper for many exams, but I'm not sure if that's a blocker at this point.

@andrewstec your current implementation as of those two commits does correctly ensure that only an instructor can access this page, but it doesn't enforce that edit permissions are required. Even if the link is only visible to editors, it can still be accessed by any instructor. I'll leave more specific comments inline.